doc/classes/EditorInspector.xml

```diff
@@ -14,6 +14,13 @@
 	<tutorials>
 	</tutorials>
 	<methods>
+		<method name="edit">
+			<return type="void" />
+			<param index="0" name="object" type="Object" />
+			<description>
+				Shows the properties of the given [param object] in the inspector for editing.
```

Comment:
Might be worth mentioning that it's intended for custom inspectors and you should use the EditorInterface's edit methods to edit in the main inspector.